src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/ApprovalsOuterApiClient.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.ProviderCommitments.Interfaces;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Infrastructure
+{
+    public class ApprovalsOuterApiClient
+    {
+        private readonly IApprovalsOuterApiHttpClient _client;
+
+        public ApprovalsOuterApiClient(IApprovalsOuterApiHttpClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<CourseDeliveryModels> GetCourseDeliveryModels(
```

Comment:
I think the name should be GetProviderCourseDeliveryModel

Reply:
Changed 👍